### DISCUSSION TITLE
Expiration of shared credentials

### DISCUSSION THREAD

**Discussion by Jademalo** [keepassxreboot/keepassxc#8364] [Q&A] [answered]
Is it possible to make is so that credentials shared with KeeShare automatically expire after a given amount of time without connection to the shared source?

I've got a portable device that I want to sync some passwords to, however I don't want to have a full duplicate of my database for security reasons.
Ideally, I want to be able to sync a specific set of passwords over to that database. KeeShare seems to be ideal for this. However, I also want these credentials to expire after a couple of days if the device hasn't been connected to the shared source. This just means that in the event the device is stolen, the passwords will expire.

Is this possible?
Thanks!

**Comment by droidmonkey** [comment, marked as answer]
No it is not possible

**Reply by Jademalo** [reply]
Ah that's a shame, thanks for the confirmation
